airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SimpleLeftSide/check.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
:thinking: Is this icon used somewhere? I can't find any reference to this in your PR?

Reply:
Leftover sorry!